lib/admin/WP_Auth0_Admin.php

```diff
@@ -209,6 +209,25 @@ public function init() {
 	 * Enqueue scripts for all Auth0 wp-admin pages
 	 */
 	public function admin_enqueue() {
+		// Register admin styles
```

Comment:
Registering these styles and scripts here lets use enqueue at any point down the line. Also setting correct dependencies to avoid the bug we're fixing in the future.